### ISSUE TITLE
Remove parts below police

### ISSUE THREAD

**Issue by ChillerDragon** [DDNetPP/maps#155]
Because these parts where contributed by others I decided on opening a discussion issue first before mapping something.
Also I do have to admit that I never played the new jail below the police. But this issue is not really about gameplay. More about feelings. I do not like how the open space below the police makes me feel while being in the police. Having no solid floor below it makes it way less cozy. The area feels polluted and exposed. I thought I will get used to it but it never happend.

## how it looks right now

![screenshot_2024-11-17_11-18-37](https://github.com/user-attachments/assets/38a9e11c-0b51-4256-9aab-c605a9d402c2)

This is how I usually farm and the lower part leaking into my view port is making me feel uncomfortable.

![screenshot_2024-11-17_11-19-07](https://github.com/user-attachments/assets/e58740b3-ebc2-41c1-b0df-c4b914c77259)

## proposed change

![screenshot_2024-11-17_11-23-06](https://github.com/user-attachments/assets/27aac4b9-b9be-44e3-a6ff-bba1f52b82c6)

It is basically reverting it to a old version. But gameplay wise this lower area had a one big positive effect in this particular part.
The issue in this flat freeze without any platforms and lots of open space is that it favors the usage of jetpack/jumps to just drag enemys without those bonuses over the open freeze. This gameplay is boring and requires no skill. It creates flow less fights. Impossible to defend against bonus strategies should be avoided by the mapper at all cost.

![screenshot_2024-11-17_11-21-47](https://github.com/user-attachments/assets/5781ce4f-f687-4993-a045-27e4a82d2d9d)

Even this pointless looking hole in the floor gives the non bonus player quite an advantage over the plain freeze floor.
It looks odd but I think it also looks good.

![screenshot_2024-11-17_11-24-19](https://github.com/user-attachments/assets/599c3ef2-5f52-45bc-ab2a-07b000d62a40)

Probably the best gameplay wise would be something like this. But in my opinion it does not look good visually (or does it? not sure yet). But gameplay wise if the non bonus player manages to get into the pit defending against flying players becomes almost a fair fight.
Also it blocks the edge jump on the left side (take speed from the right side fly over the freeze pit and land the edge on the left of the image perfectly and then jump through the entire freeze tunnel)

![screenshot_2024-11-17_11-25-30](https://github.com/user-attachments/assets/cfcce965-235a-42c0-b2f8-d0808ae819ae)

**Comment by TimePause3D** [comment]
So you just want to remove a very big part of the map just because it doesn't feel right for you? Just saying that the area below police was way before the prison was built, as well as the right wall and you suddenly feel the need to remove them all? Yeah I don't know if your feelings justify removing it all, especially when you also mentioned that you have never played on these parts before. A lot of people got used to it and love playing around it

**Comment by ChillerDragon** [comment]
@TimePause3D yea I know it comes of a bit sudden. But I always felt that way. And yes its all taste. And the word "feeling" is super vague I know. But to me that is what matters the most. When I play a game I care about how it looks like and how it makes me feel.

**Comment by fokkonaut** [comment]
I have to agree on the mixed feelings about that area.
But I definitely don't see it as a big problem like you do. Your own zoom levels and habits shouldn't be the ultimate criteria

**Comment by BulgarianTee** [comment]
In my opinion no changes should be made based on feelings, and the area suggested to be removed is way too valuable objectively:

 - When this part was being made, we specifically intended players with default zoom to be unable to see prison from police and vice versa, so that fights in police are visually isolated from other fights and therefore eliminating possible distractions. While it would be nice to make prison invisible for players who play zoomed out, I think the way it is right now is a good compromise, since most players play on default zoom anyways and we don't want to needlessly place prison too far away from police.

 - Even though the server is pretty empty and inactive during school days, during the summer this new part had a significant role in a lot of wars. Besides, its block gameplay was tested, both on a test server and with prank fights, and I think a lot of people would agree it's one of the best block arenas on the map.

 - The tight pathways of this area make it fun to maneuver around and also make fights against bonus users more fair, since bonus can't be utilized so easily there. Besides, it provides another way to spike somebody from police and also an easy way you can escape from enemies if being chased.
 
 TLDR: This part is too valuable objectively (in a gameplay sense) to be removed. It is fun to fight there and the extra spike + extra space easily accessible from police can be (and are) valuable in wars. It doesn't "feel off" for players using default zoom (the majority of players).
